accounts/tests/test_views.py

```diff
@@ -0,0 +1,103 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from accounts.models import Profile
+from django.urls import reverse
+
+
+
+class TestModel(TestCase):
+
+    def setUp(self):
+        self.register_url = reverse('register')
+        self.login_url = reverse('login')
+        self.account_url = reverse('account')
+        self.home_url = reverse('home')
+        self.user={
+            'username': 'john',
+            'email': 'john@john.com',
+            'password1': 'haslo1234',
+            'password2': 'haslo1234'
+        }
+        self.user_short_password={
+            'username': 'john',
+            'email': 'john@john.com',
+            'password1': 'haslo',
+            'password2': 'haslo'
+        }
+        self.user_unmatch_password={
+            'username': 'john',
+            'email': 'john@john.com',
+            'password1': 'abcd',
+            'password2': 'haslo'
+        }
+        self.user_invalid_email={
+            'username': 'john',
+            'email': 'john@1',
+            'password1': 'haslo1234',
+            'password2': 'haslo1234'
+        }
+        
+      
+        return super().setUp()
+        
+   
+    
+class RegisterTest(TestModel):
+    def test_sign_up_page(self):
+        response = self.client.get(self.register_url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "registration_login/register.html")
+
+    def test_register_user(self):
+        response = self.client.post(self.register_url, self.user)
+        self.assertEqual(response.status_code, 302)
```

Comment:
Sprawdziłbym jeszcze czy da się wyciągnąć usera z bazy po udanej rejestracji, tyczy się to też innych testów